src/components/ListItem.jsx

```diff
@@ -1,5 +1,56 @@
 import './ListItem.css';
+import { updateItem } from '../api';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+const currentTime = new Date();
+
+export function ListItem({ name, items, listToken }) {
+	const { isChecked, id, totalPurchases, dateLastPurchased } = items;
+
+	const currentDate = new Date();
+
+	if (isChecked) {
```

Comment:
It's best if line 9, `const currentDate = new Date();` goes in the `if (isChecked)` block since it's only used there.

Reply:
Thank you for the suggestion.

Reply:
Thank you!

Reply:
A reminder to put `const currentTime = new Date();` on this line :slightly_smiling_face: . This way, you get the most recent time whenever you click on the checkbox or load the list route. If you keep it before the export, it only runs once when you open the app.